apps/coscrad-frontend-e2e/src/e2e/resources/videos/video.detail.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,212 @@
+import { AggregateType } from '@coscrad/api-interfaces';
+
+const videoTitleInLanguage = `Video Title`;
+
+const buildId = (id: string) => `9b1deb4d-3b7d-4bad-9bdd-2b0d7b100${id}`;
+
+const buildAggregateCompositeIdentifier = (id: string) => ({
+    type: AggregateType.video,
+    id: buildId(id),
+});
+
+const videoAggregateCompositeIdentifier = buildAggregateCompositeIdentifier('001');
+
+const videoDetailRoute = `/Resources/Videos/${videoAggregateCompositeIdentifier}`;
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/Rexy-and-The-Egg-_3D-Dinosaur-Animation_-_-3D-Animation-_-Maya-3D.mp4';
+
+describe(`the video detail page`, () => {
+    before(() => {
+        cy.clearDatabase();
+
+        cy.seedTestUuids(10);
+
+        cy.seedDataWithCommand(`CREATE_VIDEO`, {
+            title: videoTitleInLanguage,
+            aggregateCompositeIdentifier: videoAggregateCompositeIdentifier,
+            videourl: validUrl,
+        });
+
+        cy.seedDataWithCommand(`PUBLISH_RESOURCE`, {
+            aggregateCompositeIdentifier: videoAggregateCompositeIdentifier,
+        });
+    });
+
+    beforeEach(() => {
+        cy.visit(videoDetailRoute);
+    });
+
+    describe(`when visiting the detail route with a bogus ID`, () => {
+        beforeEach(() => {
+            cy.visit(`/Resources/Videos/${buildId('987')}`);
+
+            cy.getByDataAttribute('not-found');
+        });
+    });
+
+    describe(`when the video is missing optional properties`, () => {
+        it(`should display the video title`, () => {
+            cy.contains(videoTitleInLanguage);
+        });
+    });
+
+    describe(`when the video has all properties`, () => {
```

Comment:
Your video doesn't yet have a transcript. You need to have a 
```
before(()=>{
    cy.seedDataWithCommand(...)

   cy.seedDataWithCommand(...)
...
})
```
that builds up a complete transcript. You'll need to
- `CREATE_TRANSCRIPT` (see create-transcript.command.ts for properties you might want to override)
- `ADD_PARTICIPANT_TO_TRANSCRIPT` X 3 (see add-participant-)
- `ADD_LINE_ITEM_TO_TRANSCRIPT` X 5 (see add-line-item-to-transcript.command.ts)
    - **OR** `IMPORT_LINE_ITEMS_TO_TRANSCRIPT` X 1 with 5 `items` (ask Blake how this one works if you go this route)
    - see import-line-items-to-transcript.command.ts

Reply:
This has been done.